addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.xml

```diff
@@ -105,92 +65,69 @@
-            <t t-call="mail.DiscussSidebarSubchannel.main"/>
-            <t t-set-slot="content">
-                <div class="overflow-hidden user-select-none" t-ref="floating" t-esc="thread.displayName"/>
-            </t>
-        </Dropdown>
-        <t t-else="" t-call="mail.DiscussSidebarSubchannel.main"/>
-    </t>
-
-    <t t-name="mail.DiscussSidebarSubchannel.main">
-        <li class="o-mail-DiscussSidebarSubchannel d-flex align-items-center flex-grow-1 position-relative" t-att-class="{ 'p-0': !store.discuss.isSidebarCompact }" t-ref="root">
-            <t t-if="!store.discuss.isSidebarCompact">
-                <svg t-if="props.isFirst" class="position-absolute opacity-75" style="top: -6px;" xmlns="http://www.w3.org/2000/svg" width="10" height="20" viewBox="0 0 10 20">
-                    <line x1="0" y1="3" x2="0" y2="100%" stroke="currentColor" stroke-width="3"/>
-                    <line x1="0" y1="100%" x2="100%" y2="100%" stroke="currentColor" stroke-width="3"/>
-                </svg>
-                <svg t-else="" class="position-absolute opacity-75" style="top: -20px;" xmlns="http://www.w3.org/2000/svg" width="10" height="34" viewBox="0 0 10 34">
-                    <line x1="0" y1="3" x2="0" y2="100%" stroke="currentColor" stroke-width="3"/>
-                    <line x1="0" y1="100%" x2="100%" y2="100%" stroke="currentColor" stroke-width="3"/>
-                </svg>
-            </t>
-            <button class="o-mail-DiscussSidebarChannel-subChannel o-mail-DiscussSidebar-item btn btn-secondary border-0 bg-inherit d-flex flex-grow-1 smaller rounded text-start text-truncate align-items-center" t-att-title="store.discuss.isSidebarCompact ? undefined : thread.displayName" t-on-click="(ev) => this.openThread(ev, thread)" t-att-class="{
+        <li class="o-mail-DiscussSidebarSubchannel d-flex align-items-center flex-grow-1 position-relative p-0" t-ref="root">
+            <svg t-if="props.isFirst" class="position-absolute me-1" style="top: -6px;" xmlns="http://www.w3.org/2000/svg" width="10" height="20" viewBox="0 0 10 20">
+                <line x1="0" y1="3" x2="0" y2="100%" stroke="currentColor" stroke-width="3"/>
+                <line x1="0" y1="100%" x2="100%" y2="100%" stroke="currentColor" stroke-width="3"/>
+            </svg>
+            <svg t-else="" class="position-absolute" style="top: -20px;" xmlns="http://www.w3.org/2000/svg" width="10" height="34" viewBox="0 0 10 34">
+                <line x1="0" y1="3" x2="0" y2="100%" stroke="currentColor" stroke-width="3"/>
+                <line x1="0" y1="100%" x2="100%" y2="100%" stroke="currentColor" stroke-width="3"/>
+            </svg>
+            <button class="o-mail-DiscussSidebarChannel-subChannel o-mail-DiscussSidebar-item btn btn-secondary border-0 bg-inherit d-flex flex-grow-1 smaller rounded text-start text-truncate align-items-center me-0 p-0 ps-2" t-att-title="thread.displayName" t-on-click="(ev) => this.openThread(ev, thread)" t-att-class="{
```

Comment:
Some classnames on `-subChannel` are incorrect from before diff, e.g. was `me-1 ps-1` and now it's `me-0 ps-2`. I see no reason to change it, as this is unrelated to the removal of the compact mode (and this subtle change was intentionally made after introduction of compact mode)